pom.xml

```diff
@@ -188,6 +188,7 @@
 						<configuration>
 							<!-- TODO: remove when it matches https://github.com/spring-cloud/spring-cloud-build/blob/master/spring-cloud-build-tools/src/main/resources/checkstyle.xml -->
 							<configLocation>src/checkstyle/checkstyle.xml</configLocation>
+							<suppressionsLocation>src/checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
```

Comment:
Right. This was my point about this: https://github.com/spring-cloud/spring-cloud-build/pull/79
So, that one doesn't work as we talked before: https://github.com/spring-cloud/spring-cloud-gcp/pull/1241/files#r239236261

/CC @dmitry-s